<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>824</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cargar archivo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: white;
	border-image: url(:/imagenes/assets/Image.png) 0 0 0 0 stretch stretch;
}

.QPushButton {
	color: white;
	background: #FF0000;
}

.QPushButton:hover {
	background: #FF7A00;
}

.QPushButton:pressed {
	background: #083C87;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>291</width>
      <height>183</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_ano">
       <property name="text">
        <string>Año correspondiente:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_archivo">
       <property name="text">
        <string>Archivo seleccionado:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="input_archivo"/>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Segundo Semestre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_periodo">
       <property name="text">
        <string>Período Correspondiente</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_ano"/>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>Primer Semestre</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="button_examinar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
width: 150px;
height: 32px;
border-radius: 15px;
background: #083C87;
}

.QPushButton:hover {
	background: #FF7A00;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
       </property>
       <property name="text">
        <string>Examinar</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar Plan de Asignatura</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	width: 150px;
	height: 60px;
	left: 735px;
	top: 324px;
	background: #FF7A00;
	border-radius: 15px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	width: 150px;
	height: 60px;
	left: 735px;
	top: 324px;
	background: #808285;
	border-radius: 15px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_ano">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>57</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_periodo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>102</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_archivo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>152</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
